test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/BasicIntegrationTest.cs

```diff
@@ -72,5 +73,49 @@ public void BuildEngine_CallProcess()
             Assert.NotNull(document.GetSyntaxTree());
             Assert.NotNull(document.GetIRDocument());
         }
+
+        [Fact]
+        public void SimpleTagHelpers()
```

Comment:
`TagHelpers` are a pretty massive part of the IR. I feel like they should be in their own integration test class.